casper/src/test/scala/coop/rchain/casper/helper/RhoSpec.scala

```diff
@@ -103,6 +103,13 @@ class RhoSpec(
           3,
           105L,
           ctx => DeployerIdContract.get(ctx)(_, _)
+        ),
+        SystemProcess.Definition[F](
+          "rho:test:crypto:secp256k1Sign",
```

Comment:
I wonder if we should make this available outside of RhoSpec?

Reply:
It'd be kind-of useless, if not dangerous: any usage on-chain would mean the private key used for signing is no longer private. Even if one passes an unforgrable name as private key, one can extract its hash off-chain and pass the private key as a byte array.

Reply:
What about contracts that for some reason want to use `insertSigned` on-chain? E.g. a contract that wants to update itself. Sounds like a typical DAO usecase (?).

Reply:
I think it's insufficient for the DAO usecase because of the singature being public (and thus: not secure)